pandas/core/generic.py

```diff
@@ -7825,6 +7828,49 @@ def _doc_parms(cls):
 
 """
 
+_any_also = """\
+See Also
+--------
+pandas.DataFrame.all : Return whether all elements are True \
+over requested axis."""
+
+_any_desc = """\
+Return whether any element is True over requested axis.
+
+One dimensional pandas.Series having boolean values will be returned. \
+Unlike pandas.DataFrame.all, pandas.DataFrame.any performs OR operation; \
+in other word, if any of the values along the specified axis is True, \
+pandas.DataFrame.any will return True."""
+
+_any_examples = """\
+Examples
+--------
+By default, any from an empty DataFrame is empty Series::
+
+    >> pd.DataFrame([]).any()
```

Comment:
Three `>`. Doesn't need to be indented.

Reply:
@TomAugspurger , you mean, without double-colon and three `>`, is it going to show code samples as required?